### PR TITLE
Ivanchuk/Cities

### PR DESCRIPTION
**Pull request by v-ivanchuk**

### CODE REVIEW

File: OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+            // Act
+            var result = await controller.Delete(id) as OkObjectResult;
+
+            // Assert
+            Assert.IsNull(result);
+        }
+
+        private CityDto FakeCity()
+        {
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs:228]: can be made static)

File: OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+            {
+                Id = 1,
+                Name = "Test",
+                Region = "Test",
+                District = "Test",
+            };
+        }
+
+        private IEnumerable<CityDto> FakeCities()
+        {
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs:239]: static)

File: OutOfSchool/OutOfSchool.WebApi.Tests/Services/CityServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        public async Task GetByCityName_WhenNameIsValid_ReturnsCities(string name)
+        {
+            // Arrange
+            var expected = await repository.GetByFilter(c => c.Name.Contains(name));
+
+            // Act
+            var result = await service.GetByCityName(name).ConfigureAwait(false);
+
+            // Assert
+            Assert.AreEqual(result.ToList().Count(), expected.Count());
```
Comment (yfedo) [inline OutOfSchool/OutOfSchool.WebApi.Tests/Services/CityServiceTests.cs:91]: can be result.Count()

**Review by yfedo — APPROVED** [review]

File: OutOfSchool/OutOfSchool.WebApi/Startup.cs [hunk 4]
```diff
@@ -113,6 +113,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<ITeacherService, TeacherService>();
             services.AddTransient<IUserService, UserService>();
             services.AddTransient<IWorkshopService, WorkshopService>();
+            services.AddTransient<ISocialGroupService, SocialGroupService>();
+            services.AddTransient<ICityService, CityService>();
```
Comment (OlhaHoliak) [inline OutOfSchool/OutOfSchool.WebApi/Startup.cs:117]: Maybe you can put it ordered by name in the group

**Review by OlhaHoliak — APPROVED** [review]

File: OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs (hunk 1 above)
Comment (v-ivanchuk) [reply OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs:228]: Dear @yfedo, 
Thank you for your advice.
This was added to the project.
Thank you.

File: OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs (hunk 2 above)
Comment (v-ivanchuk) [reply OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/CityControllerTests.cs:239]: Dear @yfedo, 
Thank you for your advice.
This was also added to the project.
Thank you.

File: OutOfSchool/OutOfSchool.WebApi.Tests/Services/CityServiceTests.cs (hunk 3 above)
Comment (v-ivanchuk) [reply OutOfSchool/OutOfSchool.WebApi.Tests/Services/CityServiceTests.cs]: Dear @yfedo, 
Thank you for your advice.
This was added to the project.
Thank you.

File: OutOfSchool/OutOfSchool.WebApi/Startup.cs (hunk 4 above)
Comment (v-ivanchuk) [reply OutOfSchool/OutOfSchool.WebApi/Startup.cs]: Dear @OlhaHoliak, 
Thank you for your advice.
This was changed in the project.
Thank you.

**Pull request merged**